### PR TITLE
Update technical-stack.md content

### PR DESCRIPTION
**Pull request by cbarbe**
Line 10 - Do you confirm there won't be BC breaks for the Asset Manager endpoints?

### CODE REVIEW

**Comment by NolwennP** [comment]
@camillebourjade, as it is mainly English fixes, can you review this PR?

**Comment by NolwennP** [comment]
@cbarbe, yes, the API for the Asset Manager is now officially released (not in Beta anymore), so it means that its endpoints are now considered as stable (ie, no BC breaks in the future).

File: content/guides/dam-connection/technical-stack.md [hunk 1]
```diff
@@ -32,16 +32,16 @@ There are two main ways to connect a DAM and a PIM:
 
 Both have their advantages and drawbacks.
 
-For those who are not comfortable with those different ways to connect, here is a bit of literature to learn more about them.  
-For the other ones, you can skip this part. :wink:
+For those who are not comfortable with these different ways to connect, below is a bit of literature to learn more about them.  
+For the others, you can skip this part. :wink:
```
Comment (camillebourjade) [inline content/guides/dam-connection/technical-stack.md]:
```suggestion
For the other ones, you can skip this part. :wink:
```

File: content/guides/dam-connection/technical-stack.md [hunk 2]
```diff
@@ -32,16 +32,16 @@ There are two main ways to connect a DAM and a PIM:
...
 
 ### API (REST/GraphQL)
 It is the most common way to connect with an external application.
 
 ![API Schema](../../img/guides/dam_pim-connection_api.svg)
 
 APIs are great for initialization purposes. But their main drawback lays in the fact you'll need a polling mechanism in order to monitor if something changed in the application.    
-In other words, you'll need to regularly ask which were the last updated DAM assets, in order to synchronize them with the PIM.
+In other words, you'll need to regularly ask what were the last updated DAM assets, in order to synchronize them with the PIM.
```
Comment (camillebourjade) [inline content/guides/dam-connection/technical-stack.md]:
```suggestion
In other words, you'll need to regularly ask which were the last updated DAM assets, in order to synchronize them with the PIM.
```

File: content/guides/dam-connection/technical-stack.md [hunk 3]
```diff
@@ -53,8 +53,8 @@ Then the connector will consume these messages.
 
 ### Webhooks
 
-With webhooks, the DAM will send newly created assets and updated assets values to your connector.
-It allows you to have a kind of "real-time" asset synchronization between your DAM and your PIM.
+With webhooks, the DAM will send freshly created assets and updated assets values to your connector.
```
Comment (camillebourjade) [inline content/guides/dam-connection/technical-stack.md]:
```suggestion
With webhooks, the DAM will send newly created assets and updated asset values to your connector.
```

File: content/guides/dam-connection/technical-stack.md [hunk 4]
```diff
@@ -85,9 +85,9 @@ When you use an asynchronous method to connect, you will need to [synchronize yo
 
 You should avoid having too many calls to the PIM API and process only data you need. This is why the [DAM attribute selection](/pre-requisites.html#define-the-attributes-of-your-asset-families) is really important.
```
Comment (camillebourjade) [inline content/guides/dam-connection/technical-stack.md]:
```suggestion
You should avoid having too many calls to the PIM API and process only the data you need. This is why the [DAM attribute selection](/pre-requisites.html#define-the-attributes-of-your-asset-families) is really important.
```

File: content/guides/dam-connection/technical-stack.md [hunk 5]
```diff
@@ -116,12 +116,12 @@ You should handle errors on DAM and PIM sides (unavailable server, authenticatio
 Here are the [client errors](/documentation/responses.html#client-errors) that you can have on the PIM side, with our PHP client.
 
 ::: info
-If you are [creating/updating multiple assets at once](/php-client/exception.html) the API will return a list of errors for each attribute that it couldn't create or update successfully.  
-You'll need to find a way to re-play those calls for the assets than have not been synchronized.
+If you are [creating/updating multiple assets at once](/php-client/exception.html), the API will return a list of errors for each attribute that it couldn't create or update successfully.
```
Comment (camillebourjade) [inline content/guides/dam-connection/technical-stack.md]:
```suggestion
If you are [creating/updating multiple assets at once](/php-client/exception.html), the API will return a list of errors for each attribute that couldn't be created or updated successfully.  
```

File: content/guides/dam-connection/technical-stack.md [hunk 6]
```diff
@@ -116,12 +116,12 @@ You should handle errors on DAM and PIM sides (unavailable server, authenticatio
...
 ::: info
-If you are [creating/updating multiple assets at once](/php-client/exception.html) the API will return a list of errors for each attribute that it couldn't create or update successfully.  
-You'll need to find a way to re-play those calls for the assets than have not been synchronized.
+If you are [creating/updating multiple assets at once](/php-client/exception.html), the API will return a list of errors for each attribute that it couldn't create or update successfully.  
+You'll need to find a way to run the calls again for the assets that have not been synchronized.
 :::
 
 ### Our skeleton
 
 As already stated in the [introduction](introduction.html#our-skeleton), we provide a skeleton from which you can take inspiration when writing your own connector.
```
Comment (camillebourjade) [inline content/guides/dam-connection/technical-stack.md]:
```suggestion
As already stated in the [introduction](introduction.html#our-skeleton), we provide a skeleton to inspire you when writing your own connector.   
```
